ocdskingfisherarchive/collection.py

```diff
@@ -0,0 +1,45 @@
+import json
+import os
+import tempfile
+
+
+class Collection:
+
+    def __init__(self, config, database_id, source_id, data_version):
+        self.config = config
+        self.database_id = database_id
+        self.source_id = source_id
+        self.data_version = data_version
+
+    def write_meta_data_file(self):
+        data = {
+            'database_id': self.database_id,
+        }
+        file_descriptor, filename = tempfile.mkstemp(prefix='archive', suffix='.json')
+        with open(filename, 'w') as file:
+            json.dump(data, file, indent=4)
+        os.close(file_descriptor)
+        return filename
+
+    def write_data_file(self):
+        file_descriptor, filename = tempfile.mkstemp(prefix='archive', suffix='.tar')
+        os.close(file_descriptor)
+
+        dir_to_add = self.config.directory_data + '/' + self.source_id + '/' + \
+            self.data_version.strftime("%Y%m%d_%H%M%S")
+        command1 = 'tar -cvf ' + filename + ' ' + dir_to_add
+        return1 = os.system(command1)
```

Comment:
Prefer `subprocess.run` with `check=True` https://docs.python.org/3/library/subprocess.html#subprocess.run

If you want to print output to console, use `stdout=subprocess.PIPE`. Otherwise, you can omit the `v` flag from the `tar` command.

Reply:
I had problems with this - given this is only a interim P.R. not the final one and this is a minor detail, can we leave this for now?

Reply:
Sure, I created an issue: https://github.com/open-contracting/kingfisher-archive/issues/19